platform/o.n.html.presenters.spi/manifest.mf

```diff
@@ -0,0 +1 @@
+OpenIDE-Module: generic
```

Comment:
Missing rename?

Reply:
Fixed in 1b111c539faafa58665, thanks.

The missing rename went unnoticed as this is an _OSGi module_. It's build script contains `<target name="jar"/>` and as such the JAR is never built, but rather copied according to instructions in properties: `release.external/generic-1.7.3.jar=modules/org-netbeans-html-presenters-spi.jar`